README.md

```diff
@@ -43,6 +43,41 @@ Running Edge in Docker with our recommended setup:
 docker run -it -p 3063:3063 -e STRICT=true -e UPSTREAM_URL=<yourunleashinstance> unleashorg/unleash-edge:<mostrecentversion> edge
 ```
 
+## Strict behavior (Since 19.2.0)
+
+If started with the `--strict` flag or the `STRICT` environment variable, Edge now starts with strict behavior and must
+be
+given
+tokens at startup.
+Edge will lock down access for new tokens that have a wider or different access scope than the tokens it was started
+with.
+
+E.g. if you start with one wildcard token with access to development `*:development.<somelongrandomstring` and later a
+client comes with a
+token accessing a different environment, say production `*:production.<somedifferentrandomstring>` Edge will return 403,
+because that would change the scope of access.
+
+Strict behavior still allows using tokens (both frontend and client) with narrower access than what the tokens it was
+started with has.
+
+E.g. If you start with one wildcard token with access to the development
+environment `*:development.<somelongrandomstring>` and your clients use various tokens with access to specific projects
+in the development environment, Edge will filter features to only grant access to the narrower scope.
+
+## Dynamic behavior
```

Comment:
In case we adopt https://github.com/Unleash/unleash-edge/pull/477/files#r1666502705

```suggestion
### Dynamic behavior
```